op-bindings/bindgen/utils.go

```diff
@@ -151,13 +152,12 @@ func genContractBindings(logger log.Logger, monorepoRootPath, abiFilePath, bytec
 		abigenVersion := bytes.Trim(versionBuf.Bytes(), "\n")
 
 		// Fetch expected abigen version (format: vX.Y.Z)
```

Comment:
I think its weird that `bindgen` itself is coupled to the version serialization but its nbd